jsdoc/tag-defs/deprecated.js

```diff
@@ -1,3 +1,6 @@
-module.exports = function() {
-  return { name: 'deprecated' };
-};
\ No newline at end of file
+module.exports = function(booleanTagTransform) {
+  return {
+    name: 'deprecated',
+    transforms: [ booleanTagTransform ]
```

Comment:
Deprecated is not actually a boolean jsdoc tag - https://jsdoc.app/tags-deprecated.html - it can contain a description of the deprecation.

Reply:
Whoops, my bad. Reverted!